### DISCUSSION TITLE
only bare repositoris or not

### DISCUSSION THREAD

**Discussion by wanneut** [nfdi4plants/ARC-specfication#31] [Comments and Suggestions]
There are two contradicting statements in the specification:

"This includes both bare repositories (without a checked out working copy) **and non-bare** repositories (i.e. a .git directory with one or more attached working copies). In particular, it is possible and intended to maintain ARCs on local user filesystems and via Git repository hosting services."
and 
"Removing the .git top-level subdirectory invalidates an ARC."
Since most hosting services do exactly that: Removing the .git. One or the other sentences hast to be wrong.

**Comment by mephenor** [comment]
"Removing the .git top-level subdirectory invalidates an ARC." could also be formulated a bit more specific to just target the non-bare repository on the user side. 
I think the intention here is basically that removing the history of the ARC invalidates it.

**Reply by chgarth** [reply]
Yes, that is exactly the point. This is what the sentence

> Removing the `.git` top-level subdirectory (and thereby all provenance information captured within the Git history) invalidates an ARC.

was meant to express. I've tweaked it to read (5d0965b2)

> Removing the `.git` top-level subdirectory (and thereby all provenance information captured within the Git history) **from a working copy** invalidates an ARC.

This should also address the Gitlab concern.
